catalogue/webapp/components/WorksSearchResult/WorksSearchResultV2.tsx

```diff
@@ -88,33 +87,28 @@ const WorkSearchResultV2: FC<Props> = ({ work, resultPosition }: Props) => {
             </WorkTitleHeading>
 
             <WorkInformation>
-              {productionDates.length > 0 && (
-                <LinkLabels items={[{ text: productionDates[0], url: null }]} />
-              )}
+              {productionDates.length > 0 && <>Date: {productionDates[0]}</>}
+
               {primaryContributorLabel && (
                 <>
-                  {productionDates.length > 0 && (
-                    <Space h={{ size: 'xs', properties: ['margin-right'] }}>
-                      |
-                    </Space>
-                  )}
-                  <Space>
-                    <LinkLabels items={[{ text: primaryContributorLabel }]} />
-                  </Space>
+                  {productionDates.length > 0 && <> | </>}
```

Comment:
We've done this elsewhere so it's fine for now, but this sort of "do we add a separate logic" always makes me feel icky.

In the past I've done separators with the `::before` selector, which works quite well on a `<ul>`: https://github.com/alexwlchan/alexwlchan.net/blob/39c2c398dcfd00b8bec32fb52d08bfe02726a653/src/_scss/_layout.scss#L42-L53

I don't know if something like that would also work here.

Reply:
Actually a great shout, I'll look into it.

Reply:
I've changed it but also moved in `works`' and `images`', hope that's alright, noticed they were static too!